torch/_meta_registrations.py

```diff
@@ -1638,6 +1638,11 @@ def meta_complex(real, imag):
     return real.new_empty(out_shape, dtype=corresponding_complex_dtype(real.dtype))
 
 
+@register_meta(aten.view.dtype)
+def view_dtype(self, dtype):
+    return self.new_empty(self.size(), dtype=dtype)
```

Comment:
you need to use `utils.clone_preserve_strides` here to make sure output strides are same as input